### PR TITLE
Put more interesting suggested topics on the search page

### LINKED ISSUE #1364: Modify featured search tags on /search
## Type
- ✨ Feature

## Add
Label: Quacks
https://next.wellcomecollection.org/search?query=quack+OR+quacks

Label: James Gillray
https://next.wellcomecollection.org/search?query=creator%3Ajames+gillray

Label: Botany
https://next.wellcomecollection.org/search?query=botany

Label: Optics
https://next.wellcomecollection.org/search?query=optics

Label: Sun

Remove:
Medicine
Nature

## Problem to be solved / User Story
Including more interesting categories for the 'suggested topics'

### PR DESCRIPTION
**Pull request by alexwlchan**
Fixes #1364. I was waiting for a test to run and fancied reading a bit more of your codebase, hope you don’t mind. 🙂

Screenshot:

screen shot 2017-08-21 at 11 09 47

### CODE REVIEW

**Comment by jennpb** [comment]
Did you notice the tiny boolean @alexwlchan? :nerd_face:

File: server/views/pages/search.njk [hunk 1]
```diff
@@ -33,9 +33,12 @@
           <div class="grid__cell">
             <h3 class="{{ {s:'WB6'} | fontClasses }}">Suggested Topics</h3>
             {% componentV2 'tags', {id: 'search-landing', tags: [
+              {text: 'Quacks', url: '/search?query=quack+OR+quacks', bgColor: 'turquoise-text'},
+              {text: 'James Gillray', url: '/search?query=creator%3Ajames+gillray', bgColor: 'turquoise-text'},
+              {text: 'Botany', url: '/search?query=botany', bgColor: 'turquoise-text'},
+              {text: 'Optics', url: '/search?query=optics', bgColor: 'turquoise-text'},
+              {text: 'Sun', url: '/search?query=Sun', bgColor: 'turquoise-text'},
```
Comment (davidpmccormick) [inline server/views/pages/search.njk]: is the query case-sensitive? Just curious.

Comment (alexwlchan) [inline server/views/pages/search.njk]: I think Elasticsearch queries are case-insensitive, but I honestly have no idea.

Comment (jennpb) [inline server/views/pages/search.njk]: Given that the search for `quack or quacks` and `quack OR quacks` gives the same number of results, I'm guessing insensitivity. Apols for letting my own typing quirks get in the way!

**Pull request merged**